test/presenters/notices/setup/prepare-return-forms.presenter.test.js

```diff
@@ -14,13 +14,12 @@ const RecipientsFixture = require('../../../fixtures/recipients.fixtures.js')
 const PrepareReturnFormsPresenter = require('../../../../app/presenters/notices/setup/prepare-return-forms.presenter.js')
 
 describe('Notices - Setup - Prepare Return Forms Presenter', () => {
+  const licenceRef = '123'
```

Comment:
Where possible, it makes the tests more readable and easier to understand if the values we use are representative of the actual data.

```suggestion
  const licenceRef = '01/123'
```